packages/hotjar/package.json

```diff
@@ -37,15 +37,15 @@
   },
   "sideEffects": false,
   "devDependencies": {
-    "@edx/browserslist-config": "1.5.0",
-    "@openedx/frontend-build": "14.3.1",
-    "react": "17.0.2",
-    "react-dom": "17.0.2",
-    "react-router-dom": "6.29.0"
+    "@edx/browserslist-config": "^1.5.0",
+    "@openedx/frontend-build": "^14.3.1",
+    "react": "^18.3.1",
+    "react-dom": "^18.3.1",
+    "react-router-dom": "^6.29.0"
```

Comment:
Should these packages be pinned?

Reply:
based on some recent conversations [here](https://github.com/openedx/frontend-build/pull/625#discussion_r1963996106), it's likely the convention of pinning NPM dependencies will change where we'll want to typically rely on `^`. Opting to proactively change it for this repo while we're here.

Reply:
This is good context on the newly minted convention that should work will with keeping our MFE packages up to date as well. 

I share the same sentiment that package updates can be prefixed WITH  `^..`, but would probably be worth sharing the information with other engineers since the pinned package version has been engrained in many peoples workflow.

Reply:
Definitely. Largely holding off on a broader inform until it gets into a more org-wide ADR more formally; not sure if that's something Brian was planning to take on yet; if not, we could drive that forward, too.

Regardless, it'll actually make dev easier when installing/updating deps since NPM by default uses `^`; usually takes effort to remember to undo the `^` and re-run `npm i` to pin it when doing something like `npm i @openedx/paragon@22`.